### ISSUE TITLE
Geographies is called with empty array

### ISSUE THREAD

**Issue by pronebird** [zcreativelabs/react-simple-maps#51]
Even though I pass the `geographyData` straight into component the first update happens without geographies. Subsequent updates bring in the geographies data. This is unexpected:

```
<Geographies geography={ geographyData } disableOptimization={ true }>
{(geographies, projection) => {
  assert(geographies.length !== 0);
}}
</Geographies>
```

**Comment by pronebird** [comment]
The issue is seems to be related to the fact that `fetchGeographies` calls `setState()` which runs asynchronously. I haven't verified that but looking at the code that's the most probable reason in my opinion.

Currently I workaround that with the simple `filter()` that removes the defects during mapping. I could as well check for the length of `geographies` and do nothing if it's empty.

```
<Geographies geography={ geographyData } disableOptimization={ true }>
  {(geographies, projection) => {
    // first update happens with empty geographies?
    if(geographies.length === 0) {
      return [];
    }
    this.state.visibleGeometry.map(({ id }) => (
      <Geography
        key={ id }
        geography={ geographies[id] }
        projection={ projection }
        style={ geographyStyle } />
    ));
  }}
</Geographies>
```

**Comment by zimrick** [comment]
This should be resolved with your fixes included in [v0.10.2](https://github.com/zcreativelabs/react-simple-maps/releases/tag/0.10.2).
